pkg/vpcmodel/nodesConnectivity.go

```diff
@@ -102,6 +116,10 @@ func (c *VPCConfig) getAllowedConnsPerDirection(isIngress bool, capturedNode Nod
 			continue
 		}
 		src, dst := switchSrcDstNodes(!isIngress, peerNode, capturedNode)
+		if c.deniedWithLBConnectivity(src, dst) {
```

Comment:
`subnets` connectivity is not effected by the `loadbalancer`? It seems to could be effected

Reply:
lb connectivity is not relevant in subnet mode

Reply:
The following restriction in connectivity was added:
`connections from privateIP to a destination that is not a pool member is denied. `
Please explain why this can not effect connectivity between subnets.

Reply:
at subnet mode we do not have nodes - i.e., we do not have pips

Reply:
But the connectivity in subnet mode is implied by the connectivity between nodes.
Anyways, this need not be part of this PR.
@adisos do you want me to open an issue for discussing this?